docker/nuke.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Add a quick comment to this file about what it does.

"Removes all datahub related containers and volumes, allowing you to start fresh, new instance of DataHub"

Reply:
Added